src/utils/OpenSwathWorkflow.cpp

```diff
@@ -1048,7 +1048,7 @@ namespace OpenMS
         if (tsv_writer.isActive()) { output.clear(); }
 
         // Set the MS1 chromatogram if available
-        if (!ms1_chromatograms.empty())
+        if (!ms1_chromatograms.empty() && ms1_chromatograms.find(transition_group.getTransitionGroupID()) != ms1_chromatograms.end())
```

Comment:
In #1772, empty chromatograms are not added to ms1_chromatograms. However, below, we try to access them based on the TransitionGroupId, resulting in a segfault.